include/setup.h

```diff
@@ -173,6 +173,10 @@ class Property {
 		return default_value;
 	}
 
+	void SetQueueableValue(std::string&& value);
+
+	const std::optional<std::string>& GetQueuedValue() const;
+
```

Comment:
I think you'll need to adjust the commit message and remove references to writing the "future state" of the config.

Reply:
Good catch. Removed!